sdk/storage/storage-blob/src/PageBlobRangeResponse.ts

```diff
@@ -0,0 +1,86 @@
+import { HttpResponse } from "@azure/core-http";
+import * as Models from "./generated/src/models";
+import { Range } from "./Range";
+
+export interface PageList {
+  pageRange?: Range[];
+  clearRange?: Range[];
+}
+
+export interface PageBlobGetPageRangesResponse
+  extends PageList,
+    Models.PageBlobGetPageRangesHeaders {
+  /**
+   * The underlying HTTP response.
+   */
+  _response: HttpResponse & {
+    /**
+     * The parsed HTTP response headers.
+     */
+    parsedHeaders: Models.PageBlobGetPageRangesHeaders;
+
+    /**
+     * The response body as text (string format)
+     */
+    bodyAsText: string;
+
+    /**
+     * The response body as parsed JSON or XML
+     */
+    parsedBody: PageList;
+  };
+}
+
+/**
+ * Contains response data for the getPageRangesDiff operation.
+ */
+export interface PageBlobGetPageRangesDiffResponse
+  extends PageList,
+    Models.PageBlobGetPageRangesDiffHeaders {
+  /**
+   * The underlying HTTP response.
+   */
+  _response: HttpResponse & {
+    /**
+     * The parsed HTTP response headers.
+     */
+    parsedHeaders: Models.PageBlobGetPageRangesDiffHeaders;
+
+    /**
+     * The response body as text (string format)
+     */
+    bodyAsText: string;
+
+    /**
+     * The response body as parsed JSON or XML
+     */
+    parsedBody: PageList;
+  };
+}
+
+export function rangeResponseFromModel(
```

Comment:
Would better to add comments for this function.

Reply:
Sure will add a comment 😃